tools/diagnostics/base_tech_bundle/cassandra_bundle.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import retrying
+
+import sdk_cmd
+
+from diagnostics.base_tech_bundle import BaseTechBundle
+
+logger = logging.getLogger(__name__)
+
+DEFAULT_RETRY_WAIT = 1000
+DEFAULT_RETRY_MAX_ATTEMPTS = 5
+
+
+class CassandraBundle(BaseTechBundle):
+    @retrying.retry(
+        wait_fixed=DEFAULT_RETRY_WAIT, stop_max_attempt_number=DEFAULT_RETRY_MAX_ATTEMPTS
+    )
+    def task_exec(self, task_id, cmd):
+        full_cmd = " ".join(
+            [
+                "export JAVA_HOME=$(ls -d ${MESOS_SANDBOX}/jdk*/jre/) &&",
+                "export TASK_IP=$(${MESOS_SANDBOX}/bootstrap --get-task-ip) &&",
+                "CASSANDRA_DIRECTORY=$(ls -d ${MESOS_SANDBOX}/apache-cassandra-*/) &&",
+                cmd,
+            ]
+        )
+
+        return sdk_cmd.marathon_task_exec(task_id, "bash -c '{}'".format(full_cmd))
+
+    def create_nodetool_status_file(self, task_id):
+        rc, stdout, stderr = self.task_exec(task_id, "${CASSANDRA_DIRECTORY}/bin/nodetool status")
```

Comment:
I had asked for this already in July: please do not hide stderr when something fails: https://github.com/mesosphere/dcos-commons/pull/2511#discussion_r206410196

Reply:
f532a4d23db964b11b059bc4bb23340916577860